src/pages/workspaces/workspace/analysis/cost-utils.test.js

```diff
@@ -0,0 +1,280 @@
+import { getAzurePricesForRegion } from 'src/libs/azure-utils'
+import { azureDisk, azureRuntime, galaxyDisk, galaxyRunning, getDisk, getGoogleRuntime, getJupyterRuntimeConfig } from 'src/pages/workspaces/workspace/analysis/_testData/testData'
+
+import { getCostDisplayForDisk, getCostDisplayForTool, getPersistentDiskCostMonthly, getRuntimeCost } from './cost-utils'
+import { cloudProviders, pdTypes } from './runtime-utils'
+import { toolLabels } from './tool-utils'
+
+
+describe('getCostDisplayForDisk', () => {
```

Comment:
if we get a fast-follow-up PR to improve the unit test coverage of cost-utils.js, I'm ok with getting this incremental PR merged.

Reply:
Yes, that's my intention!